cmd/keytransparency-sequencer/server.go

```diff
@@ -18,24 +18,26 @@ import (
 	"flag"
 	"net/http"
 
+	"github.com/google/keytransparency/cmd/serverutil"
+
 	"github.com/golang/glog"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials"
 	"google.golang.org/grpc/reflection"
 
-	"github.com/google/keytransparency/cmd/serverutil"
-	ktpb "github.com/google/keytransparency/core/proto/keytransparency_v1_grpc"
+	gpb "github.com/google/keytransparency/core/proto/keytransparency_v1_grpc"
+	_ "github.com/google/trillian/crypto/keys/der/proto"
```

Comment:
I'm not sure if it's a Go convention but I think anony imports like this are sometimes separated off into their own block

Reply:
However don't bother fighting goimports which has the definitive say on these things.

Reply:
There's no fighting with goimports here, which keeps imports in their own blocks.
In this change I'm trying to stick with `pb` and `gpb` for proto imports, which is a convention.